badge_server/test_badges.py

```diff
@@ -120,13 +231,22 @@ def setUp(self):
             'utils.highlighter', self.dependency_highlighter_stub)
         self._finder_patch = unittest.mock.patch(
             'utils.finder', self.deprecated_dep_finder_stub)
-        self._pkg_list_patch = unittest.mock.patch('main.configs.PKG_LIST', [
-            'apache-beam[gcp]',
-            'google-api-core',
-            'google-api-python-client',
-            'tensorflow',
-        ])
-
+        self._pkg_list_patch = unittest.mock.patch(
+            'compatibility_lib.configs.PKG_LIST', [
+                'apache-beam[gcp]',
+                'google-api-core',
+                'google-api-python-client',
+                'tensorflow',
+            ])
+        self._whitelist_urls_patch = unittest.mock.patch(
+            'compatibility_lib.configs.WHITELIST_URLS', {
+                'git+git://github.com/google/apache-beam.git':
+                'apache-beam[gcp]',
```

Comment:
This needs to be indented

http://google.github.io/styleguide/pyguide.html#34-indentation

Reply:
Thanks! I don't know why yapf did this - maybe I forgot to use --style=google?